mne/viz/_3d.py

```diff
@@ -577,6 +577,8 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
         show EEG sensors in their digitized locations or projected onto the
         scalp, or a list of these options including ``[]`` (equivalent of
         False).
+    fwd : instance of Forward
+        The forward solution.
```

Comment:
```suggestion
        The forward solution. If present, will plot the source normals,
        and use ``fwd['src']`` as the ``src`` argument.
```

.. and also change the code so that it actually does the second thing (uses `fwd['src']` as `src`).

Reply:
... but if that looks ugly because the arrows + src plotting looks bad, then we don't need to add that behavior. But please keep the bit about saying it's used to plot the source normals.

Reply:
I would not do that as I want to be able to pass src and fwd to see which dipole was dropped due to min_dist param